src/components/SubscriptionsPage.vue

```diff
@@ -25,12 +25,12 @@
         >
             <span v-text="group.groupName !== '' ? group.groupName : $t('video.all')" />
             <div v-if="group.groupName != '' && selectedGroup == group">
-                <font-awesome-icon class="mx-2" icon="edit" @click="showEditGroupModal = true" />
-                <font-awesome-icon class="mx-2" icon="circle-minus" @click="deleteGroup(group)" />
+                <i class="unocss-icon fa6 fa6-edit mx-2" @click="showEditGroupModal = true"></i>
+                <i class="unocss-icon fa6 fa6-circle-minus mx-2" @click="deleteGroup(group)"></i>
```

Comment:
You must use the right unocss classes. To use a fa edit icon, you must use i-fa6:edit for example.